server.js

```diff
@@ -326,10 +400,10 @@ function SocketIOFileUploadServer() {
 				} else {
 					// If we're not saving the file, we are ready to start receiving data now.
 					if (!self.dir) {
-						socket.emit("siofu_ready", {
+						socket.emit(_getTopicName("_ready"), _wrapData({
 							id: data.id,
 							name: null
-						});
+						}, "reday"));
```

Comment:
ready, not reday

This typo should be caught by a test.

Reply:
Fixed